cmd/start-amazon-cloudwatch-agent/start-amazon-cloudwatch-agent.go

```diff
@@ -37,13 +37,21 @@ var (
 
 	translatorBinaryPath string
 	agentBinaryPath      string
+	agentMode            string
```

Comment:
I prefer to have all the changes located inside config-translator. For example, you could change this function for config-translator: https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/util/sdkutil.go#L29-L45. You can check the environment variable here and then return ec2 mode.

Reply:
Moved the env check to config-translator